spec/integration/qt_spec.cr

```diff
@@ -15,6 +15,37 @@ describe "Qt-specific wrapper features" do
           called.should be_true
         end
 
+        context "overloaded signals" do
+          it "creates methods for each overload" do
+            subject = Test::SomeObject.new
+
+            overload = 0
+            subject.on_overloaded(Int32) do |_i|
```

Comment:
Leading-underscore named variables mean they're unused.  Please simply remove the underscore for these variables.  Otherwise LGTM 👍